components/drawer/drawer.component.ts

```diff
@@ -77,7 +77,7 @@ const NZ_CONFIG_MODULE_NAME: NzConfigKey = 'drawer';
             <div class="ant-drawer-wrapper-body" [style.height]="isLeftOrRight ? '100%' : null">
               <div
                 *ngIf="nzTitle || nzClosable"
-                [class.ant-drawer-header]="!!nzTitle"
+                class="ant-drawer-header"
                 [class.ant-drawer-header-close-only]="!nzTitle"
               >
```

Comment:
Since we are planning to sync ant-design's DOM structure, there should be adding one more wrapping div with class `ant-drawer-header-title`

Reply:
Do you mean, the ng-zorro and ant-design will share the same DOM structure in the feature?

Reply:
Ok, has changed the DOM`s structure